### ISSUE TITLE
Build an app to generate photorealistic faces using TensorFlow and Streamlit

### ISSUE THREAD

**Issue by khuyentran1401** [khuyentran1401/machine-learning-articles#5]
## TL;DR
Interactively synthesize faces with simple techniques
![image](https://miro.medium.com/max/1400/0*kqTo76YSl92YkYDV)
### Article Link
https://towardsdatascience.com/build-an-app-to-synthesize-photorealistic-faces-using-tensorflow-and-streamlit-dd2545828021

### Author
Adrien Treuille

## Key Takeaways
* Make interactive models with Streamlit
* Synthesize face with TensorFlow
* Use GAN to change the faces

## Useful Tools
*  Streamlit
*  TensorFlow
* GAN

## Comments/ Questions
* Would it be easy and interesting to use Streamlit to create our own application for our machine learning model?
